packages/rocketchat-api/server/v1/integrations.js

```diff
@@ -5,9 +5,9 @@ RocketChat.API.v1.addRoute('integrations.create', { authRequired: true }, {
 			name: String,
 			enabled: Boolean,
 			username: String,
-			urls: [String],
+			urls: Match.Maybe([String]),
 			channel: String,
-			event: String,
+			event: Match.Maybe(String),
```

Comment:
This is absolutely required for outgoing webhooks, where is the check for this now?